include/lsst/afw/detection/Bootprint.h

```diff
@@ -74,6 +74,10 @@ class Bootprint : public lsst::daf::base::Citizen,
     explicit Bootprint(std::shared_ptr<geom::SpanSet> inputSpans,
                        afw::table::Schema const & peakSchema,
                        geom::Box2I const & region=geom::Box2I());
+    explicit Bootprint(): lsst::daf::base::Citizen(typeid(this)),
+                          _spans(std::make_shared<geom::SpanSet>()),
+                          _peaks(PeakTable::makeMinimalSchema()),
+                          _region(geom::Box2I()) {}
```

Comment:
Could the other constructor that just uses `PeakTable::makeMinimalSchema()` initialize it the same way, or does it have to use `setPeakSchema`?  Probably best to make them consistent, and if this approach works, it's the better one.